mappings/net/minecraft/block/Block.mapping

```diff
@@ -23,6 +23,8 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 		ARG 3 side
 	METHOD method_23349 getVelocityMultiplier ()F
 	METHOD method_23350 getJumpVelocityMultiplier ()F
+	METHOD method_27839 isBlock (Lnet/minecraft/class_2248;)Z
+		ARG 1 block
```

Comment:
This seems like a weird name... What does an `isBlock(Block)` method do?

Reply:
Checks if the block equals the block in parameter. Should I rename it to `equals`?

Reply:
why can't they use `==`?

Reply:
Yeah they just uses `this == block` in implementation. I guess this is their effort to move away from hardcoding; and I personally suggest just calling the method `is`.